### PR TITLE
Support NIE as username

### LINKED ISSUE #5: JSON::ParserError
I'm receiving a `JSON::ParserError` when using this gem:

```
require 'bankscrap'
require 'bankscrap-bbva'

bbva = Bankscrap::BBVA::Bank.new(user: '......', password: '.......')

puts bbva.accounts.inspect
```

Here is the error output:
```
/usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/json/common.rb:156:in `parse': 743: unexpected token at '<logon/> (JSON::ParserError)
'
	from /usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/json/common.rb:156:in `parse'
	from /usr/local/lib/ruby/gems/2.4.0/gems/bankscrap-bbva-2.0.1/lib/bankscrap/bbva/bank.rb:49:in `fetch_accounts'
	from /usr/local/lib/ruby/gems/2.4.0/gems/bankscrap-2.0.5/lib/bankscrap/bank.rb:30:in `initialize'
	from /usr/local/lib/ruby/gems/2.4.0/gems/bankscrap-bbva-2.0.1/lib/bankscrap/bbva/bank.rb:20:in `initialize'
	from run.rb:4:in `new'
	from run.rb:4:in `<main>'
```

### PR DESCRIPTION
**Pull request by gitviola**
Fixes #5

### CODE REVIEW

File: lib/bankscrap/bbva/bank.rb [hunk 1]
```diff
@@ -108,14 +108,19 @@ def fetch_transactions_for(account, start_date: Date.today - 1.month, end_date:
 
       # As far as we know there are two types of identifiers BBVA uses
```
Comment (javiercr) [inline lib/bankscrap/bbva/bank.rb]: could you change this `two` for `three`? :)

**Comment by javiercr** [comment]
Awesome, this is exactly the kind of contributions we need! Thanks 💙!

**Comment by gitviola** [comment]
@javiercr I missed that `two`. Now the comments should be perfect :)

**Comment by javiercr** [comment]
Merged and published to rubygems as `v2.0.2`, thank you!

**Pull request merged**